src/config/config.ts

```diff
@@ -35,12 +35,13 @@ const conf = {
     /** Options in refresh */
     pollInterval: {
       0: 'Pause',
-      5000: '5 sec',
-      10000: '10 sec',
-      15000: '15 sec',
-      30000: '30 sec',
-      60000: '1 min',
-      300000: '5 min'
+      5000: 'Every 5 sec',
```

Comment:
This makes the graph refresh duplicate "Every": it's now showing like "Every Every 15 sec" :)

Reply:
Hahhaa I'll cahnge it ^^